app/models/profile_field.rb

```diff
@@ -22,7 +22,7 @@ class ProfileField < ApplicationRecord
   private
 
   def generate_attribute_name
-    self.attribute_name = label.titleize.scan(WORD_REGEX).join.underscore
+    self.attribute_name = "attribute_#{SecureRandom.uuid.delete('-')}"
```

Comment:
This line is the core change - how do we generate the attribute/method name for a newly created profile field. Everything else here is updating our assumptions about that decision that leaked into the rest of the app.